vms/secp256k1fx/camino_credential.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2023, Chain4Travel AG. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package secp256k1fx
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils"
+	"github.com/ava-labs/avalanchego/utils/crypto"
+)
+
+var errMSigNotUniqueOrSorted = errors.New("multisig aliases not sorted or unique")
+
+type CredentialIntf interface {
+	Signatures() [][crypto.SECP256K1RSigLen]byte
+	Verify() error
+}
+
+type MultisigCredential struct {
+	Credential      `serialize:"true"`
+	MultiSigAliases []ids.ShortID `serialize:"true" json:"multisigAliases"`
```

Comment:
Is this multisig aliases or just addresses from one multisig?

Reply:
Only aliases, a list of aliases which should be checked, see PR description.
There can be wired constellations where verification could fail (theoretically, but possible)
Lets say we have a1, M1{a2, a3 => T2}, a2 => T2 and you provide signatures a1 and a2.
Without knowing that we shouldn't ignore M1, the verification would fail.